### DISCUSSION TITLE
Handling of hanging degrees of freedom

### DISCUSSION THREAD

**Discussion by moroz3339** [mfem/mfem#3423] [Q&A]
Hello everyone!
I`m gonna implement octree on a voxel grid and due to it occurs a problem with hanging degrees of freedom. How does MFEM handle a hanging degree of freedom? Does it make linear interpolating?
Thanks!
![conflicting_constraints](https://user-images.githubusercontent.com/113342974/212670836-5e993238-b263-42cd-aee1-aefa98e34019.png)

**Comment by v-dobrev** [comment]
You can read about non-conforming meshes in MFEM here: https://doi.org/10.1137/18M1193992.
